libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizer.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.Bot.Builder.AI.Luis
     public class LuisRecognizer : ITelemetryRecognizer
     {
         [JsonProperty("$kind")]
-        public const string DeclarativeType = "Microsoft.LuisRecognizer";
+        public const string Kind = "Microsoft.LuisRecognizer";
```

Comment:
Breaking change.

Reply:
good catch, changing back.

---
In reply to: [411524785](https://github.com/microsoft/botbuilder-dotnet/pull/3758#discussion_r411524785) [](ancestors = 411524785)